compiler/rustc_borrowck/src/borrow_set.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(rustc::untranslatable_diagnostic)]
+#![deny(rustc::diagnostic_outside_of_impl)]
```

Comment:
Could you deny it in the whole `rustc_borrowck` crate in `lib.rs` instead, if that works already? If it doesn't you can still remove at least some of the denies in submodules. `#![deny]` attribute applies transitively for all submodules as well.

Reply:
there would be still 200 more errors messages if I deny it in lib..

I just trimmed submodules, did I get it right?